### ISSUE TITLE
Microsoft edge problem with finding the address bar when you start

### ISSUE THREAD

**Issue by Nikita34196** [nvaccess/nvda#10967]
this problem has recently emerged. 1. Open Microsoft Edge. in the settings at launch, you should be set to open a new tab.. 2. Write something in the address bar and click input. Expected. nvda goes to the search engine page with results. Actually. nvda stays in the address bar and has to go through all the browser panels to get to the web page. windows 10 19603 nvda version alpha-19938,e8cde2ca edge 83.0.478.1 (Официальная сборка) canary (64-разрядная версия)

**Comment by Adriani90** [comment]
@gdata1 I cannot reproduce this in last stable Edge Chromium 81 with NVDA Version: alpha-20015,56a62097.

I did following:
1. Open a new tab
2. Press alt+d to go to the address bar
3. Typed google.com
Actual behavior: NVDA jumped to the webpage, landing on the google search field. This is also expected behavior.
Can you confirm with the versions above?

**Comment by gdata1** [comment]
@Adriani90 Hello. I have this happening on version 84.0.488.0 (Official Assembly) canary (64-bit version) this happens even when I open the browser settings page. nvda remains in the address line. NVDA version alpha-20005,14c2e2ec

**Comment by Adriani90** [comment]
@gdata1 not sure if this will have any effect, but could you please try to run the COM registration fixing tool in NVDA menu under "tools"?
So it seems this problem might have appeared after Edge Cromium 81. Or can you also reproduce this problem in Edge Cromium 81? If not, I think a bug with Microsoft should be filled as well.

**Comment by gdata1** [comment]
@Adriani90 I did what you said, but it didn't help. the problem started on version 83. with all versions of nvda. including nvda alpha and nvda 2019.3.

**Comment by Adriani90** [comment]
@Nikita34196 how is it now with NVDA 2023.1 and Edge 112? Make sure to enable "switch to browse mode after loading the page" in NVDA's browse mode settings.

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since many years. I am closing this issue as abandoned. please feel free to comment on this if you are still having the issue and we can reopen.
